packages/atomic/src/components/atomic-sort-dropdown/atomic-sort-dropdown.tsx

```diff
@@ -122,17 +123,19 @@ export class AtomicSortDropdown implements InitializableComponent {
       <div class="relative">
         <select
           id={this.id}
-          class="flex-grow appearance-none rounded bg-background text-primary font-bold border border-neutral py-3 pl-4 pr-24"
+          class="h-10 flex-grow cursor-pointer appearance-none rounded bg-background border border-neutral pl-3 pr-24 focus:outline-none focus:border-primary-light focus:text-primary-light hover:border-primary-light hover:text-primary-light"
```

Comment:
I'm wondering how this will scale in the future.

I'd be interested (in some internal refactor/cleanup stage for atomic) if we could "analyse" the frequency for each classes that we repeat on many many components/section.

And if it would make sense to extract more utilities (I think it's called? in tailwind world).

Or if it would just make things more complicated.

Reply:
Yes it surely feels that way sometimes, in facets we did a good job at reusing functional components, classes, etc.
For these buttons, I've opened https://coveord.atlassian.net/browse/KIT-910 but theres is going to be more possibilities of reusing classes. There is also a few classes already in global.pcss